README.md

```diff
@@ -108,17 +108,16 @@ fn run() -> Result<(), Error> {
 ```rust,no_run
 use elasticsearch::{
     auth::Credentials,
-    Error, Elasticsearch, 
+    Error, Elasticsearch,
     http::transport::Transport,
 };
-use url::Url;
 
-fn run() -> Result<(), Error> {
+fn main() -> Result<(), Error> {
     let cloud_id = "cluster_name:Y2xvdWQtZW5kcG9pbnQuZXhhbXBsZSQzZGFkZjgyM2YwNTM4ODQ5N2VhNjg0MjM2ZDkxOGExYQ==";
     // can use other types of Credentials too, like Bearer or ApiKey
     let credentials = Credentials::Basic("<username>".into(), "<password>".into());
     let transport = Transport::cloud(cloud_id, credentials)?;
-    let client = Elasticsearch::new(transport);
+    let _client = Elasticsearch::new(transport);
```

Comment:
I realize client isn’t used, but would prefer to keep this as `client`, in line with all examples.